images.go

```diff
@@ -179,6 +102,17 @@ var Images = []Image{
 			},
 		},
 	},
+	Image{
+		// see https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml
+		Name: "gcr.io/google_containers/fluentd-elasticsearch",
```

Comment:
For the record, I changed this from `gcr.io/google-containers/fluentd-elasticsearch` to `gcr.io/google_containers/fluentd-elasticsearch` - note the change from a dash to an underscore.

It took me ages to notice, and it changes the alphabetisation (and both urls work)